2015-02-12  Jakub Jelinek  <jakub@redhat.com>

	PR c++/64884
	* g++.dg/tm/pr47573.C: Only run on comdat_group effective targets.

--- gcc/testsuite/g++.dg/tm/pr47573.C.jj	2011-11-08 23:35:11.000000000 +0100
+++ gcc/testsuite/g++.dg/tm/pr47573.C	2015-02-12 14:26:23.873151135 +0100
@@ -1,4 +1,7 @@
-// { dg-do compile }
+// Without comdat support, we don't see the body of the
+// extern template class constructor, so limit this to
+// known comdat targets.
+// { dg-do compile { target comdat_group } }
 // { dg-options "-fgnu-tm" }
 
 template<typename _Tp> class allocator
